### PR TITLE
Toolbars, Icons, Reinitialization, translations and metadata update

### PR DESCRIPTION
**Pull request by PatrykPaziewski**

### CODE REVIEW

File: VX_integration.py [hunk 1]
```diff
@@ -548,52 +580,65 @@ def DownloadVxData(self):
...
                     self.DownloadVxData()
-                    self.dlg.pushButton_2.setEnabled(True)
-                    self.dlg.reinitialize.setEnabled(True)
-                    self.i += 1
-        else:
-            self.show_error(self.tr("Connection failed! - Please try again"))
+            
+        self.dlg.pushButton_2.setEnabled(True)
+        self.dlg.reinitialize.setEnabled(True)
+        self.actions[2].setEnabled(True)
```
Comment (van9konst) [inline VX_integration.py]: please check text inside action, and not index in array

File: VX_integration.py [hunk 2]
```diff
@@ -548,52 +580,65 @@ def DownloadVxData(self):
...
+            self.dlg.pushButton_2.setEnabled(True)
+            self.dlg.reinitialize.setEnabled(True)
+            self.actions[2].setEnabled(True)
+            self.actions[3].setEnabled(True)
+            self.actions[1].setChecked(True)
+        
+        else:
+            self.dlg.pushButton_2.setEnabled(False)
+            self.dlg.reinitialize.setEnabled(False)
+            self.actions[2].setEnabled(False)
```
Comment (van9konst) [inline VX_integration.py]: please check text inside action, and not index in array

File: VX_integration.py [hunk 3]
```diff
@@ -651,4 +689,3 @@ def run(self):
         if result:
```
Comment (van9konst) [inline VX_integration.py]: not used?

File: VX_integration.py [hunk 4]
```diff
@@ -13,7 +13,7 @@
 
 from PyQt5.QtCore import QSettings, QTranslator, qVersion, QCoreApplication, QTimer, QVariant, QThread, pyqtSignal, QThreadPool, QRunnable, QObject
 from PyQt5.QtGui import QIcon, QPixmap, QMovie
-from PyQt5.QtWidgets import QAction, QWidget, QTableWidgetItem, QDialogButtonBox, QProgressBar
+from PyQt5.QtWidgets import QAction, QWidget, QTableWidgetItem, QDialogButtonBox, QProgressBar, QToolBar
```
Comment (van9konst) [inline VX_integration.py]: please separate imports 
https://www.python.org/dev/peps/pep-0008/#id23

**Review by van9konst — CHANGES_REQUESTED** [review]

File: VX_integration.py (hunk 1 above)
Comment (PatrykPaziewski) [reply VX_integration.py]: Not able to - action do not have a specific name

**Review by van9konst — APPROVED** [review]

**Pull request merged**